mne/tests/test_label.py

```diff
@@ -692,6 +692,8 @@ def test_stc_to_label():
         assert_labels_equal(l1, l2, decimal=4)
 
 
+@requires_nibabel()
+@requires_dipy()
```

Comment:
a test that did need dipy now needs it? this will break code. You should avoid this.